src/main/java/com/example/service/SimpleService.java

```diff
@@ -0,0 +1,74 @@
+package com.example.service;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.commons.codec.digest.DigestUtils;
+import org.springframework.stereotype.Service;
+
+import com.example.domain.User;
+import com.example.domain.User.Role;
+
+@Service
+public class SimpleService {
+	private Set<String> tokens;
+	private Map<String, User> users;
+	
+	public SimpleService() {
+		users = new HashMap<String, User>();
+		tokens = new HashSet<String>();
+		
+		users.put("admin", new User("admin", DigestUtils.sha256Hex("adminpass"), Role.ADMIN));
+	}
+
+	public Set<String> getTokens() {
```

Comment:
You shouldn't have getters and setters in a spring bean/service type class in most of the cases.